configuration/ConfigHandler.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef CONFIG_CONFIGHANDLER_HPP
+#define CONFIG_CONFIGHANDLER_HPP
+
+#include <string>
+#include <map>
+
+namespace Configuration
+{
+
+    /**
+     * @brief Handles the server configuration
+     * by parsing the config.yml file
+     */
+    class ConfigHandler
+    {
+    public:
+        ConfigHandler(std::string);
+        ~ConfigHandler();
+
+        std::string getIP(void);
+        std::string getMotd(void);
+        std::string getPort(void);
+        std::string getMaxPlayers(void);
```

Comment:
Same there with the max player count, why is it an `std::string` and not an int ?